fluent-bundle/src/resolver.ts

```diff
@@ -139,6 +172,7 @@ function VariableReference(scope, {name}) {
       if (arg instanceof Date) {
         return new FluentDateTime(arg.getTime());
       }
+    // eslint-disable-next-line no-fallthrough
```

Comment:
Is this one really needed here. From my understanding it shouldn't because the `return` prevents fallthrough?

Reply:
The `return` statement is guarded by an `if`, and the fall-through is desired here: if it's an object but not a `Date`, fall through to `default` and report an error.